cmd/dev.go

```diff
@@ -0,0 +1,22 @@
+package cmd
+
+import (
+	"github.com/okteto/okteto/cmd/up"
+	"github.com/spf13/cobra"
+)
+
+func Dev() *cobra.Command {
+	command := &cobra.Command{
+		Use:   "dev [up|down|status|doctor|exec|restart]",
+		Short: "Manage your development environment",
+	}
+
+	command.AddCommand(up.Up())
+	command.AddCommand(Down())
+	command.AddCommand(Status())
+	command.AddCommand(Doctor())
```

Comment:
Shouldn't it be the other way around? I updated the help of the commands from root to show this:
```
$ okteto up
Command "up" is deprecated, use `dev` instead
```

Reply:
it won't be deprecated, it's a valid alias.
What I mean is that if you do `okteto dev --help` I would be specific about the option to run `okteto up` as an alias of `okteto dev up`

Reply:
Sorry, I misunderstood. I've already fixed it and it looks like this:
```
lokteto dev --help                                                                                 ──(Thu,Apr07)─┘
Manage your development environment

Usage:
  okteto dev [command]

Available Commands:
  doctor      Generate a zip file with the okteto logs. You can use `okteto doctor [svc]` as an alias
  down        Deactivate your development container. You can use `okteto down [svc]` as an alias
  endpoints   Show endpoints for an environment. You can use `okteto endpoints` as an alias
  exec        Execute a command in your development container. You can use `okteto exec <command>` as an alias
  restart     Restart the deployments listed in the services field of your okteto manifest. You can use `okteto restart [svc]` as an alias
  status      Status of the synchronization process. You can use `okteto status` as an alias
  up          Launch your development environment. You can use `okteto up [svc]` as an alias

```

If you want some custom phrasing in these commands, let me know and I can customize them individually

Reply:
@rberrelleza @RinkiyaKeDad how would you show this information?

Reply:
If `okteto dev up` is the main command and `okteto up` is an alias, I think we should remove mentioning aliases in the help for `okteto dev` and just have command descriptions there - because it is feeling kind of repetitive reading similar sentences in the help. 

Instead, when someone runs `okteto up --help`, we could then show them the message that `okteto up` is just an alias for `okteto dev up`. 

Were you preferring it how it currently is?

Reply:
I think that part of the problem was the clutter in `okteto --help`, where we would get all of this `dev` commands with their descriptions. I like what you said Arsh but I think it goes against the idea of removing the `dev` subcommands from the root `okteto --help`

Reply:
@AgustinRamiroDiaz is it possible to show the alias info only on `okteto dev up --help` for example?
Yes, they idea is to reduce the number of commands when you do `okteto --help`

Reply:
@pchico83 yes it is! I'll work on it